editor/scene_tree_dock.cpp

```diff
@@ -4140,14 +4140,16 @@ SceneTreeDock::SceneTreeDock(Node *p_scene_root, EditorSelection *p_editor_selec
 	ED_SHORTCUT("scene_tree/delete", TTR("Delete"), Key::KEY_DELETE);
 
 	button_add = memnew(Button);
-	button_add->set_flat(true);
+	button_add->set_flat(false);
```

Comment:
```suggestion
```
Button is non-flat by default, so no need to set it. (same in other Buttons)

Reply:
Ah got it, I'll address this too. For future reference, is it always better to remove redundant lines like this when possible? I was under the impression that in some cases it might be better to explicitly set stuff so the intent is clear, but maybe that's not needed for something this common